supervision/annotators/core.py

```diff
@@ -1050,6 +1055,7 @@ def annotate(
         detections: Detections,
         labels: Optional[List[str]] = None,
         custom_color_lookup: Optional[np.ndarray] = None,
+        custom_text_color_lookup: Optional[np.ndarray] = None,
```

Comment:
Let's use the same `custom_color_lookup`.

Reply:
ok